packaging/ovirt-web-ui.war/WEB-INF/branding.jsp

```diff
@@ -0,0 +1,129 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
```

Comment:
WEB-INF allows to hide files form direct access. If we move it one level up, there would be a need to come up with something smart to return for `GET /web-ui/branding.jsp` and additional unnecessary code in `branging.jsp`. This way only `/branding/*` is handled. So I'd rather keep it here.